examples/cpp-simple/server.cc

```diff
@@ -16,101 +16,115 @@
+              << std::endl;
+    std::cout << "GameServer Update, state: " << gameserver.status().state()
+              << std::endl;
+  });
 }
 
 int main() {
-    std::cout << "C++ Game Server has started!" << std::endl;
-
-    std::cout << "Getting the instance of the SDK!" << std::endl;
-    agones::SDK *sdk = new agones::SDK();
-
-    std::cout << "Attempting to connect..." << std::endl;
-    bool connected = sdk->Connect();
-    if (!connected) {
-        std::cout << "Could not connect to the sidecar. Exiting!" << std::endl;
-        return -1;
-    }
-    std::cout << "...handshake complete." << std::endl;
-
-    std::thread health (doHealth, sdk);
-    std::thread watch (watchUpdates, sdk);
-
-    std::cout << "Setting a label" << std::endl;
-    grpc::Status status = sdk->SetLabel("test-label", "test-value");
-    if (!status.ok()) {
-        std::cout << "Could not run SetLabel(): "+ status.error_message() + ". Exiting!" << std::endl;
-        return -1;
-    }
-
-    std::cout << "Setting an annotation" << std::endl;
-    status = sdk->SetAnnotation("test-annotation", "test value");
-    if (!status.ok()) {
-        std::cout << "Could not run SetAnnotation(): "+ status.error_message() + ". Exiting!" << std::endl;
+  std::cout << "C++ Game Server has started!" << std::endl;
+
+  std::cout << "Getting the instance of the SDK!" << std::endl;
+  agones::SDK *sdk = new agones::SDK();
+
+  std::cout << "Attempting to connect..." << std::endl;
+  bool connected = sdk->Connect();
+  if (!connected) {
+    std::cout << "Could not connect to the sidecar. Exiting!" << std::endl;
+    return -1;
+  }
+  std::cout << "...handshake complete." << std::endl;
+
+  std::thread health(doHealth, sdk);
+  std::thread watch(watchUpdates, sdk);
+
+  std::cout << "Setting a label" << std::endl;
+  grpc::Status status = sdk->SetLabel("test-label", "test-value");
+  if (!status.ok()) {
+    std::cout << "Could not run SetLabel(): " + status.error_message() +
+                     ". Exiting!"
+              << std::endl;
+    return -1;
+  }
+
+  std::cout << "Setting an annotation" << std::endl;
+  status = sdk->SetAnnotation("test-annotation", "test value");
+  if (!status.ok()) {
+    std::cout << "Could not run SetAnnotation(): " + status.error_message() +
+                     ". Exiting!"
+              << std::endl;
+    return -1;
+  }
+
+  std::cout << "Marking server as ready..." << std::endl;
+  status = sdk->Ready();
+  if (!status.ok()) {
+    std::cout << "Could not run Ready(): " + status.error_message() +
+                     ". Exiting!"
+              << std::endl;
+    return -1;
+  }
+
+  std::cout << "...marked Ready" << std::endl;
+
+  std::cout << "Getting GameServer details..." << std::endl;
+  agones::dev::sdk::GameServer gameserver;
+  status = sdk->GameServer(&gameserver);
+
+  if (!status.ok()) {
+    std::cout << "Could not run GameServer(): " + status.error_message() +
```

Comment:
Since this code is already using an output stream, avoid the string concatenation (`op+`) and instead just stream each piece, like `std::cout << "Could not run GameServer(): " << status.error_message() << " Exiting!\n";`

Reply:
I fixed this here (and elsewhere) while I was updating the newline change.